open-metadata-implementation/view-services/server-author-view/server-author-view-server/src/main/java/org/odpi/openmetadata/viewservices/serverauthor/handlers/ServerAuthorViewHandler.java

```diff
@@ -573,7 +573,7 @@ public void configureEngineService(String className, String methodName, String s
             EngineHostConfigurationClient client = new EngineHostConfigurationClient(this.userId,
                                                                                      serverToBeConfiguredName,
                                                                                      this.platformURL);
-            client.configureEngineService(serviceURLMarker, engineServiceOptions, engines);
+            client.configureEngineService(null, null, serviceURLMarker, engineServiceOptions, engines);
```

Comment:
I wonder why  this method was not overridden, 2  parameters of null at the start of the arg list seems strange.

Reply:
Partner URL and ServerName are mandatory.   However, I do not want to break the UI and we are not able to invest in it at the moment.  It continues to work as well as it did before.